### ISSUE TITLE
Build Failed since Xcode 12 Using Swift Package Manager

### ISSUE THREAD

**Issue by kbw2204** [ReactorKit/ReactorKit#168]
I have same Issue when I using reactorKit with SPM
```
The package product 'WeakMapTable' requires minimum platform version 9.0 for the iOS platform, 
but this target supports 8.0
```

but the build error disappeared when I installed it through cocoaPod without installing it as a SPM.
However, I get new error in the view specific, and eventually I downgraded to Xcode 11.7. ㅠ _ㅜ..
```
failed to demangle superclass of ConcatSink from mangled name 'I\272'
2020-09-21 17:45:28.708889+0900 ProjectName[1699:244374] failed to demangle superclass of ConcatSink from mangled name '\^BI\M-:\^H'
```

**Comment by dodgecm** [comment]
This is fixed in #163 which hasn't been merged yet, but you can use my ReactorKit fork in the meantime if you need an immediate fix.

**Comment by kbw2204** [comment]
@dodgecm I checked and the build works fine!☺️ without any error! Thanks for your feedback!
